locales/base/translation.json

```diff
@@ -2185,5 +2185,17 @@
     "discover": {
       "tabName": "Discover"
     }
+  },
+  "jumpstartStatus": {
+    "loading": {
+      "title": "Your funds are on their way",
+      "description": "The funds that your friend sent you are being sent over using crypto magic. Please wait..."
```

Comment:
lol can we update the copy? i was told that the copy was finalised in figma last week